client/package.json

```diff
@@ -5,15 +5,22 @@
   "dependencies": {
     "@material-ui/core": "^4.3.3",
     "@material-ui/icons": "^4.2.1",
+    "babel": "^6.23.0",
+    "babel-jest": "^24.9.0",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
     "history": "^4.9.0",
+    "jest-enzyme": "^7.1.1",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
     "react-flexbox-grid": "^2.1.2",
     "react-hook-form": "^3.22.3",
     "react-redux": "^7.1.1",
     "react-router-dom": "^5.0.1",
     "react-scripts": "3.0.1",
+    "react-test-renderer": "^16.9.0",
     "redux": "^4.0.4",
+    "redux-mock-store": "^1.5.3",
```

Comment:
Test packages should be added as a dev dependency with `npm uninstall <package> && npm install -D <package>` or manually by editing the `package.json`